recipes-ni/initscripts-nilrt/files/populateconfig

```diff
@@ -25,7 +25,6 @@ then
 		fi
 	fi
 
-	mkdir -p /etc/natinst/share/tpm
```

Comment:
Can you do a codesearch and attempt to identify if this path is used by NI packages?

Reply:
It is only used by nilrtdiskcrypt.